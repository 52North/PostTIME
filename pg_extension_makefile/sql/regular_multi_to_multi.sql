SELECT pt_regular_multi_to_multi('R2/2000/P1Y/P9Y');
SELECT pt_regular_multi_to_multi('R200/2000/P1Y/P9Y');
SELECT pt_regular_multi_to_multi('R10/2013-4-30T12:12:59.999/P9Y8M7DT6H5M4.321S');
SELECT pt_regular_multi_to_multi('R10/2013-4-30T12:12:59.999/P9Y8M7DT6H5M4.321S/PT12H22M59.123S');