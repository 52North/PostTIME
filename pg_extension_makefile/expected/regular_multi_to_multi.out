SELECT pt_regular_multi_to_multi('R2/2000/P1Y/P9Y');
                                                                  pt_regular_multi_to_multi                                                                  
-------------------------------------------------------------------------------------------------------------------------------------------------------------
 CAL0012000-01-01T00:00:00.000Z/2001-01-01T00:00:00.000Z,2010-01-01T00:00:00.000Z/2011-01-01T00:00:00.000Z,2020-01-01T00:00:00.000Z/2021-01-01T00:00:00.000Z
(1 row)

SELECT pt_regular_multi_to_multi('R200/2000/P1Y/P9Y');
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        pt_regular_multi_to_multi                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 CAL0012000-01-01T00:00:00.000Z/2001-01-01T00:00:00.000Z,2010-01-01T00:00:00.000Z/2011-01-01T00:00:00.000Z,2020-01-01T00:00:00.000Z/2021-01-01T00:00:00.000Z,2030-01-01T00:00:00.000Z/2031-01-01T00:00:00.000Z,2040-01-01T00:00:00.000Z/2041-01-01T00:00:00.000Z,2050-01-01T00:00:00.000Z/2051-01-01T00:00:00.000Z,2060-01-01T00:00:00.000Z/2061-01-01T00:00:00.000Z,2070-01-01T00:00:00.000Z/2071-01-01T00:00:00.000Z,2080-01-01T00:00:00.000Z/2081-01-01T00:00:00.000Z,2090-01-01T00:00:00.000Z/2091-01-01T00:00:00.000Z,2100-01-01T00:00:00.000Z/2101-01-01T00:00:00.000Z,2110-01-01T00:00:00.000Z/2111-01-01T00:00:00.000Z,2120-01-01T00:00:00.000Z/2121-01-01T00:00:00.000Z,2130-01-01T00:00:00.000Z/2131-01-01T00:00:00.000Z,2140-01-01T00:00:00.000Z/2141-01-01T00:00:00.000Z,2150-01-01T00:00:00.000Z/2151-01-01T00:00:00.000Z,2160-01-01T00:00:00.000Z/2161-01-01T00:00:00.000Z,2170-01-01T00:00:00.000Z/2171-01-01T00:00:00.000Z,2180-01-01T00:00:00.000Z/2181-01-01T00:00:00.000Z,2190-01-01T00:00:00.000Z/2191-01-01T00:00:00.000Z,2200-01-01T00:00:00.000Z/2201-01-01T00:00:00.000Z,2210-01-01T00:00:00.000Z/2211-01-01T00:00:00.000Z,2220-01-01T00:00:00.000Z/2221-01-01T00:00:00.000Z,2230-01-01T00:00:00.000Z/2231-01-01T00:00:00.000Z,2240-01-01T00:00:00.000Z/2241-01-01T00:00:00.000Z,2250-01-01T00:00:00.000Z/2251-01-01T00:00:00.000Z,2260-01-01T00:00:00.000Z/2261-01-01T00:00:00.000Z,2270-01-01T00:00:00.000Z/2271-01-01T00:00:00.000Z,2280-01-01T00:00:00.000Z/2281-01-01T00:00:00.000Z,2290-01-01T00:00:00.000Z/2291-01-01T00:00:00.000Z,2300-01-01T00:00:00.000Z/2301-01-01T00:00:00.000Z,2310-01-01T00:00:00.000Z/2311-01-01T00:00:00.000Z,2320-01-01T00:00:00.000Z/2321-01-01T00:00:00.000Z,2330-01-01T00:00:00.000Z/2331-01-01T00:00:00.000Z,2340-01-01T00:00:00.000Z/2341-01-01T00:00:00.000Z,2350-01-01T00:00:00.000Z/2351-01-01T00:00:00.000Z,2360-01-01T00:00:00.000Z/2361-01-01T00:00:00.000Z,2370-01-01T00:00:00.000Z/2371-01-01T00:00:00.000Z,2380-01-01T00:00:00.000Z/2381-01-01T00:00:00.000Z,2390-01-01T00:00:00.000Z/2391-01-01T00:00:00.000Z,2400-01-01T00:00:00.000Z/2401-01-01T00:00:00.000Z,2410-01-01T00:00:00.000Z/2411-01-01T00:00:00.000Z,2420-01-01T00:00:00.000Z/2421-01-01T00:00:00.000Z,2430-01-01T00:00:00.000Z/2431-01-01T00:00:00.000Z,2440-01-01T00:00:00.000Z/2441-01-01T00:00:00.000Z,2450-01-01T00:00:00.000Z/2451-01-01T00:00:00.000Z,2460-01-01T00:00:00.000Z/2461-01-01T00:00:00.000Z,2470-01-01T00:00:00.000Z/2471-01-01T00:00:00.000Z,2480-01-01T00:00:00.000Z/2481-01-01T00:00:00.000Z,2490-01-01T00:00:00.000Z/2491-01-01T00:00:00.000Z,2500-01-01T00:00:00.000Z/2501-01-01T00:00:00.000Z,2510-01-01T00:00:00.000Z/2511-01-01T00:00:00.000Z,2520-01-01T00:00:00.000Z/2521-01-01T00:00:00.000Z,2530-01-01T00:00:00.000Z/2531-01-01T00:00:00.000Z,2540-01-01T00:00:00.000Z/2541-01-01T00:00:00.000Z,2550-01-01T00:00:00.000Z/2551-01-01T00:00:00.000Z,2560-01-01T00:00:00.000Z/2561-01-01T00:00:00.000Z,2570-01-01T00:00:00.000Z/2571-01-01T00:00:00.000Z,2580-01-01T00:00:00.000Z/2581-01-01T00:00:00.000Z,2590-01-01T00:00:00.000Z/2591-01-01T00:00:00.000Z,2600-01-01T00:00:00.000Z/2601-01-01T00:00:00.000Z,2610-01-01T00:00:00.000Z/2611-01-01T00:00:00.000Z,2620-01-01T00:00:00.000Z/2621-01-01T00:00:00.000Z,2630-01-01T00:00:00.000Z/2631-01-01T00:00:00.000Z,2640-01-01T00:00:00.000Z/2641-01-01T00:00:00.000Z,2650-01-01T00:00:00.000Z/2651-01-01T00:00:00.000Z,2660-01-01T00:00:00.000Z/2661-01-01T00:00:00.000Z,2670-01-01T00:00:00.000Z/2671-01-01T00:00:00.000Z,2680-01-01T00:00:00.000Z/2681-01-01T00:00:00.000Z,2690-01-01T00:00:00.000Z/2691-01-01T00:00:00.000Z,2700-01-01T00:00:00.000Z/2701-01-01T00:00:00.000Z,2710-01-01T00:00:00.000Z/2711-01-01T00:00:00.000Z,2720-01-01T00:00:00.000Z/2721-01-01T00:00:00.000Z,2730-01-01T00:00:00.000Z/2731-01-01T00:00:00.000Z,2740-01-01T00:00:00.000Z/2741-01-01T00:00:00.000Z,2750-01-01T00:00:00.000Z/2751-01-01T00:00:00.000Z,2760-01-01T00:00:00.000Z/2761-01-01T00:00:00.000Z,2770-01-01T00:00:00.000Z/2771-01-01T00:00:00.000Z,2780-01-01T00:00:00.000Z/2781-01-01T00:00:00.000Z,2790-01-01T00:00:00.000Z/2791-01-01T00:00:00.000Z,2800-01-01T00:00:00.000Z/2801-01-01T00:00:00.000Z,2810-01-01T00:00:00.000Z/2811-01-01T00:00:00.000Z,2820-01-01T00:00:00.000Z/2821-01-01T00:00:00.000Z,2830-01-01T00:00:00.000Z/2831-01-01T00:00:00.000Z,2840-01-01T00:00:00.000Z/2841-01-01T00:00:00.000Z,2850-01-01T00:00:00.000Z/2851-01-01T00:00:00.000Z,2860-01-01T00:00:00.000Z/2861-01-01T00:00:00.000Z,2870-01-01T00:00:00.000Z/2871-01-01T00:00:00.000Z,2880-01-01T00:00:00.000Z/2881-01-01T00:00:00.000Z,2890-01-01T00:00:00.000Z/2891-01-01T00:00:00.000Z,2900-01-01T00:00:00.000Z/2901-01-01T00:00:00.000Z,2910-01-01T00:00:00.000Z/2911-01-01T00:00:00.000Z,2920-01-01T00:00:00.000Z/2921-01-01T00:00:00.000Z,2930-01-01T00:00:00.000Z/2931-01-01T00:00:00.000Z,2940-01-01T00:00:00.000Z/2941-01-01T00:00:00.000Z,2950-01-01T00:00:00.000Z/2951-01-01T00:00:00.000Z,2960-01-01T00:00:00.000Z/2961-01-01T00:00:00.000Z,2970-01-01T00:00:00.000Z/2971-01-01T00:00:00.000Z,2980-01-01T00:00:00.000Z/2981-01-01T00:00:00.000Z,2990-01-01T00:00:00.000Z/2991-01-01T00:00:00.000Z,3000-01-01T00:00:00.000Z/3001-01-01T00:00:00.000Z,3010-01-01T00:00:00.000Z/3011-01-01T00:00:00.000Z,3020-01-01T00:00:00.000Z/3021-01-01T00:00:00.000Z,3030-01-01T00:00:00.000Z/3031-01-01T00:00:00.000Z,3040-01-01T00:00:00.000Z/3041-01-01T00:00:00.000Z,3050-01-01T00:00:00.000Z/3051-01-01T00:00:00.000Z,3060-01-01T00:00:00.000Z/3061-01-01T00:00:00.000Z,3070-01-01T00:00:00.000Z/3071-01-01T00:00:00.000Z,3080-01-01T00:00:00.000Z/3081-01-01T00:00:00.000Z,3090-01-01T00:00:00.000Z/3091-01-01T00:00:00.000Z,3100-01-01T00:00:00.000Z/3101-01-01T00:00:00.000Z,3110-01-01T00:00:00.000Z/3111-01-01T00:00:00.000Z,3120-01-01T00:00:00.000Z/3121-01-01T00:00:00.000Z,3130-01-01T00:00:00.000Z/3131-01-01T00:00:00.000Z,3140-01-01T00:00:00.000Z/3141-01-01T00:00:00.000Z,3150-01-01T00:00:00.000Z/3151-01-01T00:00:00.000Z,3160-01-01T00:00:00.000Z/3161-01-01T00:00:00.000Z,3170-01-01T00:00:00.000Z/3171-01-01T00:00:00.000Z,3180-01-01T00:00:00.000Z/3181-01-01T00:00:00.000Z,3190-01-01T00:00:00.000Z/3191-01-01T00:00:00.000Z,3200-01-01T00:00:00.000Z/3201-01-01T00:00:00.000Z,3210-01-01T00:00:00.000Z/3211-01-01T00:00:00.000Z,3220-01-01T00:00:00.000Z/3221-01-01T00:00:00.000Z,3230-01-01T00:00:00.000Z/3231-01-01T00:00:00.000Z,3240-01-01T00:00:00.000Z/3241-01-01T00:00:00.000Z,3250-01-01T00:00:00.000Z/3251-01-01T00:00:00.000Z,3260-01-01T00:00:00.000Z/3261-01-01T00:00:00.000Z,3270-01-01T00:00:00.000Z/3271-01-01T00:00:00.000Z,3280-01-01T00:00:00.000Z/3281-01-01T00:00:00.000Z,3290-01-01T00:00:00.000Z/3291-01-01T00:00:00.000Z,3300-01-01T00:00:00.000Z/3301-01-01T00:00:00.000Z,3310-01-01T00:00:00.000Z/3311-01-01T00:00:00.000Z,3320-01-01T00:00:00.000Z/3321-01-01T00:00:00.000Z,3330-01-01T00:00:00.000Z/3331-01-01T00:00:00.000Z,3340-01-01T00:00:00.000Z/3341-01-01T00:00:00.000Z,3350-01-01T00:00:00.000Z/3351-01-01T00:00:00.000Z,3360-01-01T00:00:00.000Z/3361-01-01T00:00:00.000Z,3370-01-01T00:00:00.000Z/3371-01-01T00:00:00.000Z,3380-01-01T00:00:00.000Z/3381-01-01T00:00:00.000Z,3390-01-01T00:00:00.000Z/3391-01-01T00:00:00.000Z,3400-01-01T00:00:00.000Z/3401-01-01T00:00:00.000Z,3410-01-01T00:00:00.000Z/3411-01-01T00:00:00.000Z,3420-01-01T00:00:00.000Z/3421-01-01T00:00:00.000Z,3430-01-01T00:00:00.000Z/3431-01-01T00:00:00.000Z,3440-01-01T00:00:00.000Z/3441-01-01T00:00:00.000Z,3450-01-01T00:00:00.000Z/3451-01-01T00:00:00.000Z,3460-01-01T00:00:00.000Z/3461-01-01T00:00:00.000Z,3470-01-01T00:00:00.000Z/3471-01-01T00:00:00.000Z,3480-01-01T00:00:00.000Z/3481-01-01T00:00:00.000Z,3490-01-01T00:00:00.000Z/3491-01-01T00:00:00.000Z,3500-01-01T00:00:00.000Z/3501-01-01T00:00:00.000Z,3510-01-01T00:00:00.000Z/3511-01-01T00:00:00.000Z,3520-01-01T00:00:00.000Z/3521-01-01T00:00:00.000Z,3530-01-01T00:00:00.000Z/3531-01-01T00:00:00.000Z,3540-01-01T00:00:00.000Z/3541-01-01T00:00:00.000Z,3550-01-01T00:00:00.000Z/3551-01-01T00:00:00.000Z,3560-01-01T00:00:00.000Z/3561-01-01T00:00:00.000Z,3570-01-01T00:00:00.000Z/3571-01-01T00:00:00.000Z,3580-01-01T00:00:00.000Z/3581-01-01T00:00:00.000Z,3590-01-01T00:00:00.000Z/3591-01-01T00:00:00.000Z,3600-01-01T00:00:00.000Z/3601-01-01T00:00:00.000Z,3610-01-01T00:00:00.000Z/3611-01-01T00:00:00.000Z,3620-01-01T00:00:00.000Z/3621-01-01T00:00:00.000Z,3630-01-01T00:00:00.000Z/3631-01-01T00:00:00.000Z,3640-01-01T00:00:00.000Z/3641-01-01T00:00:00.000Z,3650-01-01T00:00:00.000Z/3651-01-01T00:00:00.000Z,3660-01-01T00:00:00.000Z/3661-01-01T00:00:00.000Z,3670-01-01T00:00:00.000Z/3671-01-01T00:00:00.000Z,3680-01-01T00:00:00.000Z/3681-01-01T00:00:00.000Z,3690-01-01T00:00:00.000Z/3691-01-01T00:00:00.000Z,3700-01-01T00:00:00.000Z/3701-01-01T00:00:00.000Z,3710-01-01T00:00:00.000Z/3711-01-01T00:00:00.000Z,3720-01-01T00:00:00.000Z/3721-01-01T00:00:00.000Z,3730-01-01T00:00:00.000Z/3731-01-01T00:00:00.000Z,3740-01-01T00:00:00.000Z/3741-01-01T00:00:00.000Z,3750-01-01T00:00:00.000Z/3751-01-01T00:00:00.000Z,3760-01-01T00:00:00.000Z/3761-01-01T00:00:00.000Z,3770-01-01T00:00:00.000Z/3771-01-01T00:00:00.000Z,3780-01-01T00:00:00.000Z/3781-01-01T00:00:00.000Z,3790-01-01T00:00:00.000Z/3791-01-01T00:00:00.000Z,3800-01-01T00:00:00.000Z/3801-01-01T00:00:00.000Z,3810-01-01T00:00:00.000Z/3811-01-01T00:00:00.000Z,3820-01-01T00:00:00.000Z/3821-01-01T00:00:00.000Z,3830-01-01T00:00:00.000Z/3831-01-01T00:00:00.000Z,3840-01-01T00:00:00.000Z/3841-01-01T00:00:00.000Z,3850-01-01T00:00:00.000Z/3851-01-01T00:00:00.000Z,3860-01-01T00:00:00.000Z/3861-01-01T00:00:00.000Z,3870-01-01T00:00:00.000Z/3871-01-01T00:00:00.000Z,3880-01-01T00:00:00.000Z/3881-01-01T00:00:00.000Z,3890-01-01T00:00:00.000Z/3891-01-01T00:00:00.000Z,3900-01-01T00:00:00.000Z/3901-01-01T00:00:00.000Z,3910-01-01T00:00:00.000Z/3911-01-01T00:00:00.000Z,3920-01-01T00:00:00.000Z/3921-01-01T00:00:00.000Z,3930-01-01T00:00:00.000Z/3931-01-01T00:00:00.000Z,3940-01-01T00:00:00.000Z/3941-01-01T00:00:00.000Z,3950-01-01T00:00:00.000Z/3951-01-01T00:00:00.000Z,3960-01-01T00:00:00.000Z/3961-01-01T00:00:00.000Z,3970-01-01T00:00:00.000Z/3971-01-01T00:00:00.000Z,3980-01-01T00:00:00.000Z/3981-01-01T00:00:00.000Z,3990-01-01T00:00:00.000Z/3991-01-01T00:00:00.000Z,4000-01-01T00:00:00.000Z/4001-01-01T00:00:00.000Z
(1 row)

SELECT pt_regular_multi_to_multi('R10/2013-4-30T12:12:59.999/P9Y8M7DT6H5M4.321S');
                                                                                                                                pt_regular_multi_to_multi                                                                                                                                 
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 CAL0012013-04-30T12:12:59.999Z,2023-01-06T18:18:04.320Z,2032-09-14T00:23:08.641Z,2042-05-21T06:28:12.962Z,2052-01-28T12:33:17.283Z,2061-10-05T18:38:21.604Z,2071-06-13T00:43:25.925Z,2081-02-20T06:48:30.246Z,2090-10-27T12:53:34.567Z,2100-07-04T18:58:38.888Z,2110-03-12T01:03:43.208Z
(1 row)

SELECT pt_regular_multi_to_multi('R10/2013-4-30T12:12:59.999/P9Y8M7DT6H5M4.321S/PT12H22M59.123S');
                                                                                                                                                                                                                                                                          pt_regular_multi_to_multi                                                                                                                                                                                                                                                                          
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 CAL0012013-04-30T12:12:59.999Z/2023-01-06T18:18:04.320Z,2023-01-07T06:41:03.443Z/2032-09-14T12:46:07.764Z,2032-09-15T01:09:06.887Z/2042-05-22T07:14:11.208Z,2042-05-22T19:37:10.331Z/2052-01-30T01:42:14.652Z,2052-01-30T14:05:13.775Z/2061-10-07T20:10:18.096Z,2061-10-08T08:33:17.218Z/2071-06-15T14:38:21.539Z,2071-06-16T03:01:20.662Z/2081-02-23T09:06:24.983Z,2081-02-23T21:29:24.106Z/2090-10-31T03:34:28.427Z,2090-10-31T15:57:27.550Z/2100-07-08T22:02:31.871Z,2100-07-09T10:25:30.994Z/2110-03-16T16:30:35.315Z,2110-03-17T04:53:34.438Z/2119-11-24T10:58:38.759Z
(1 row)

SELECT pt_regular_multi_to_multi('TCS002R10/1000.0000/P10/P1');
                                                                                                                           pt_regular_multi_to_multi                                                                                                                           
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 TCS0021000.000000/1010.000000,1011.000000/1021.000000,1022.000000/1032.000000,1033.000000/1043.000000,1044.000000/1054.000000,1055.000000/1065.000000,1066.000000/1076.000000,1077.000000/1087.000000,1088.000000/1098.000000,1099.000000/1109.000000,1110.000000/1120.000000
(1 row)

SELECT pt_regular_multi_to_multi('TCS002R10/1000.0000/P10');
                                                         pt_regular_multi_to_multi                                                         
-------------------------------------------------------------------------------------------------------------------------------------------
 TCS0021000.000000,1010.000000,1020.000000,1030.000000,1040.000000,1050.000000,1060.000000,1070.000000,1080.000000,1090.000000,1100.000000
(1 row)

